roles/static_inventory/tasks/sshtun.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Create ssh tunnel systemd service
+  template:
+    src: ssh-tunnel.service.j2
+    dest: /etc/systemd/system/ssh-tunnel.service
+    mode: 0755
+
+- name: reload the systemctl daemon after file update
+  shell: systemctl daemon-reload
```

Comment:
This can be a `command` instead. Shell is for when you need pipes or io redirection.